### ISSUE TITLE
Yogi Serve - file system permission error

### ISSUE THREAD

**Issue by solmsted** [yui/yogi#30]
I think this is related to the fix for issue #21.  Yogi tries to auto update yuidoc.json.  With a default install of npm, `npm -g install yogi` requires root access, so updating yuidoc.json also requires root access.

I got this in the terminal

```
[fedora@solmsted gallery-alea]$ yogi serve
yogi [info]  using yogi@0.0.48 on node@0.8.15
yogi [info]  using module: gallery-alea
yogi [info]  adding tests route

fs.js:338
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: EACCES, permission denied '/usr/local/lib/node_modules/yogi/configs/yuidoc/yuidoc.json'
    at Object.fs.openSync (fs.js:338:18)
    at Object.fs.openSync (/usr/local/lib/node_modules/yogi/node_modules/istanbul/node_modules/fileset/node_modules/glob/node_modules/graceful-fs/graceful-fs.js:68:26)
    at Object.fs.openSync (/usr/local/lib/node_modules/yogi/node_modules/rimraf/node_modules/graceful-fs/graceful-fs.js:68:26)
    at Object.fs.openSync (/usr/local/lib/node_modules/yogi/node_modules/cpr/node_modules/graceful-fs/graceful-fs.js:68:26)
    at Object.fs.writeFileSync (fs.js:756:15)
    at mods.yuidoc (/usr/local/lib/node_modules/yogi/lib/cmds/serve.js:123:24)
    at mods.version (/usr/local/lib/node_modules/yogi/lib/cmds/serve.js:143:17)
    at Request.mod.callback [as _callback] (/usr/local/lib/node_modules/yogi/lib/api.js:50:17)
    at Request.init.self.callback (/usr/local/lib/node_modules/yogi/node_modules/request/main.js:122:22)
    at Request.EventEmitter.emit (events.js:99:17)
```

Is there a way to chmod during install or update to relax the write permissions on that file?

**Comment by davglass** [comment]
Can you install from master to test this out?

**Comment by solmsted** [comment]
sure, I'll try it now.

fyi, I fixed the file permissions manually and ran into the same issue with the file /usr/local/lib/node_modules/yogi/configs/selleck/project.json.  I fixed that one too and it's working.

---

GitHub wouldn't let me add another comment so I've updated this one.

Yes, yogi serve no longer throws errors and I didn't have to chmod the files.
but I'm not sure if the files are updating as intended
